src/config/config.ts

```diff
@@ -205,6 +205,7 @@ export class Configuration {
 
   crypto = {
     fee: 0.012,
+    feeUsedRef: 0.011,
```

Comment:
I would propose to switch to a refBonus of 0.1% instead. So the fee, when using a ref code would be `crypto.fee - crypto.refBonus` (see buy fee).

Reply:
resulting from discussion yesterday evening, changed calculation to be similar to buy routes and renamed config entry to `refBonus` and added `refBonus` to be delivered with crypto routes